flipperdevices/flipperzero-firmware
# [FL-2828] Dolphin score update take 2

# What's new

- Improved score counting for SubGHz, NFC, LFRFID, iButton and Infrared apps

# Verification 

1. Compile and upload
2. Open a serial terminal connected to Flipper's debug port
3. Use the aforementioned applications. Observe score changes in the debug output.

All eligible actions should be rewarded according to the documentation.

# Checklist (For Reviewer)
